prisma/schema.prisma

```diff
@@ -121,6 +121,8 @@ model VideoMetadata {
   migrated_video_1080p_mp4_1 String?
   migrated_video_360p_mp4_1  String?
   migrated_video_720p_mp4_1  String?
+  timestamp Int?
```

Comment:
Same here. `timestamp` does not reflect the correct data by its name.

Reply:
Okay so I made a mistake in data type.  `DateTime` will be appropriate right?